bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```
@@ -497,6 +497,10 @@ PreferencePageParameterValues_pageLabelSeparator = \ >\
 ThemingEnabled = E&nable theming
 ThemeChangeWarningText = Restart for the theme changes to take full effect
 ThemeChangeWarningTitle = Theme Changed
+RuntimeSettingChangeWarningTitle = DPI Setting Changed
+RuntimeSettingChangeWarningDesc = Restart for the effect to take place
+UpdateUIZoomTitle = Update UI scaling on runtime
```

May it be more self-explaining when putting the `monitor-specific` property in the label? E.g.
```suggestion
UpdateUIZoomTitle = Monitor-specific UI scaling
```